client/src/components/Banner.tsx

```diff
@@ -1,25 +1,20 @@
 import React from 'react';
 import Button from 'plaid-threads/Button';
-import PropTypes from 'prop-types';
 
 const PLAID_ENV = process.env.REACT_APP_PLAID_ENV;
 
-const propTypes = {
-  initialSubheading: PropTypes.bool,
-};
-
-const defaultProps = {
-  initialSubheading: false,
-};
+interface Props {
+  initialSubheading?: boolean;
+}
 
-const Banner = ({ initialSubheading }) => {
+const Banner = (props: Props) => {
   const initialText =
-    'This is an example app that outlines an end-to-end integration with Plaid.';
+    'This is an example app that shows a Plaid integration for a Personal Finance Manager Use Case.';
```

Comment:
"This example app demonstrates how to use/integrate Plaid for a personal finance management use case."

Could use "use" "integrate" or some other word above ^, just thought we could make this sentences a tad stronger.

Reply:
done!